src/components/dictionaryConcepts/components/ViewConceptMappings.jsx

```diff
@@ -13,11 +13,22 @@ class ViewConceptMappings extends Component {
     });
   };
 
+  handleMappings = (mappings) => {
+    const newMappings = [];
+    mappings.forEach((mapping) => {
+      if (mapping.retired === false) {
+        newMappings.push(mapping);
+      }
+    });
+    return newMappings;
+  }
+
   render() {
     const { editModalIsOpen } = this.state;
     const {
       mappings, displayName, showDeleteMappingModal, handleDeleteMapping, source,
     } = this.props;
+    const myMappings = this.handleMappings(mappings);
```

Comment:
Instead of `myMappings` , you could use a more descriptive name like `activeMappings` or anything appropriate

Reply:
working on it